.eslintrc.yml

```diff
@@ -62,3 +62,21 @@ rules:
     - off
   import/no-default-export:
     - error
+  import/newline-after-import:
```

Comment:
keep order of rules

we have something like this
- common off
- common
- typescript off
- typescript
- import off
- import